assets/js/WfParameter.js

```diff
@@ -274,14 +274,16 @@ export default class WfParameter extends React.Component {
         />
       )
     } else if (id_name == 'rename-entries') {
+      const changeLoadAll = ((val) => {this.props.setParamText('display-all', val)});
```

Comment:
Move this up so that it's a property of the `WfParameter`, the way `paramChanged` is. Give it as verbose a name as it needs -- `setParamText_DisplayAll()`, for instance.

Here's why: where it is now, each time we reach this line we create a new `changeLoadAll` function, which means we change that prop on `RenameEntries`. The prop change forces a `render()`.

There are many ways to avoid a render. By far the simplest and most widely-used is to move `changeLoadAll` to the class level. (Yes, it makes `WfParameter.js` more terrible. But that's because `WfParameter` is inescapably terrible.) Then we can make `RenameEntries` extend `React.PureComponent` and presto! No more re-renders.